### DISCUSSION TITLE
Is it possible for Renovatebot to create a PR to a custom, dynamic branch?

### DISCUSSION THREAD

**Discussion by alvindizon** [renovatebot/renovate#18151] [Request Help] [answered]
Let me expound on what I mean by "custom and dynamic branch"--let's say our team that uses Bitbucket currently works on `release/1.0.0` branch and when we are done for that release, we merge it to `master`. Then we cut another release branch, `release/1.0.1`, and then we work on that branch. Would it be possible for renovatebot to be configured in a such a way that it will make PRs that point to the latest `release` branch, which in my example would be `release/1.0.1`?

**Comment by rarkins** [comment, marked as answer]
Not currently possible. You would need #5027

**Reply by alvindizon** [reply]
I see, thanks for clarifying!
